web/src/server_events_dispatch.js

```diff
@@ -681,7 +675,6 @@ export function dispatch_normal_event(event) {
                             message_lists.current.update_trailing_bookend(true);
                         }
                     }
-                    message_live_update.rerender_messages_view();
```

Comment:
We might need to do something like this if we're using a archived channel icon for messages in that channel. But maybe we need to decide that in the broader design conversation on what to do when a channel is archived or you lose access to it.

Reply:
Well, this is the lose-access case. I think there, it might make sense to rerender to visually disappear messages from a combined feed or similar view that you lost access to?

Not sure, that might require a bit more systematic of an approach, like also deleting all messages in the channel from `message_store` and similar structures too.